src/db/mysql/pool.rs

```diff
@@ -70,6 +71,9 @@ impl MysqlDbPool {
         let manager = ConnectionManager::<MysqlConnection>::new(settings.database_url.clone());
         let builder = Pool::builder()
             .max_size(settings.database_pool_max_size.unwrap_or(10))
+            .connection_timeout(Duration::from_secs(
+                settings.database_pool_max_size.unwrap_or(30) as u64,
```

Comment:
Kinda think we should use a different setting for this rather than overload what pool max size.

Reply:
Ah, looks like this is a typo. I see `database_pool_max_size` defined later.